src/scss/_admonitions.scss

```diff
@@ -149,12 +149,12 @@
   }
 
   &.note {
-    border-color: rgba(var(--color-admonition-info), 1);
+    border-color: rgba(var(--color-admonition-primary), 1);
```

Comment:
I would prefer that we keep this at "info" category, but so we should maybe combine the info/primary categories (or at least give them the same colors, see the "third issue" in https://github.com/pandas-dev/pydata-sphinx-theme/issues/279#issuecomment-722404488)